cypress/e2e/pages/governance/create/index.spec.ts

```diff
@@ -0,0 +1,705 @@
+      });
+      cy.findByTestId(
+        "Governance.Create.Step1.TextArea.Cycles.Increment"
+      ).click();
+      cy.findByTestId("Governance.Create.Step1.TextArea.Cycles").within(() => {
+        cy.get("textarea").should("have.text", "2");
+      });
+      cy.findByTestId(
+        "Governance.Create.Step1.TextArea.Cycles.Decrement"
+      ).click();
+      cy.findByTestId("Governance.Create.Step1.TextArea.Cycles").within(() => {
+        cy.get("textarea").should("have.text", "1");
+      });
+    });
+
+    it("should have step 1 `Receiving Address` input for CFP proposal creation", () => {
+      cy.findByTestId(
+        "Governance.Create.Step1.TextArea.ReceivingAddress"
+      ).should("be.visible");
+
+      cy.findByTestId(
+        "Governance.Create.Step1.TextArea.ReceivingAddress"
+      ).within(() => {
+        cy.get("textarea").click().type(receivingAddress);
+        cy.get("Governance.Create.Step1.TextArea.ErrorMsg").should("not.exist");
+      });
+
+      cy.findByTestId("Governance.Create.Step1.ReviewProposal").should(
+        "be.disabled"
+      );
+
+      cy.findByTestId(
+        "Governance.Create.Step1.TextArea.ReceivingAddress"
+      ).within(() => {
+        cy.findByTestId(
+          "Governance.Create.Step1.TextArea.ReceivingAddress.ClearForm"
+        ).click();
+        cy.findByTestId("Governance.Create.Step1.TextArea.ErrorMsg").should(
+          "have.text",
+          "Invalid payout address. Only DFI addresses are accepted"
+        );
+      });
+    });
+
+    it("should be able to clear input using clear form button in step 1", () => {
+      cy.findByTestId(
+        "Governance.Create.Step1.TextArea.ReceivingAddress"
+      ).within(() => {
+        cy.get("textarea").click().type(receivingAddress);
+      });
+      cy.findByTestId("Governance.Create.Step1.ClearForm").click();
+      cy.findByTestId(
+        "Governance.Create.Step1.TextArea.ReceivingAddress"
+      ).within(() => {
+        cy.get("textarea").should("have.text", "");
+      });
+    });
+
+    it("should be able to fill in CFP proposal inputs in step 1 and Review Proposal", () => {
+      cy.findByTestId("Governance.Create.Step1.TextArea.NameOfProposal").within(
+        () => {
+          cy.get("textarea").click().type(nameOfProposal);
+        }
+      );
+      cy.findByTestId("Governance.Create.Step1.TextArea.Discussion").within(
+        () => {
+          cy.get("textarea").click().type(discussion);
+        }
+      );
+      cy.findByTestId(
+        "Governance.Create.Step1.TextArea.AmountRequested"
+      ).within(() => {
+        cy.get("textarea").click().type(amountRequested);
+      });
+      cy.findByTestId(
+        "Governance.Create.Step1.TextArea.ReceivingAddress"
+      ).within(() => {
+        cy.get("textarea").click().type(receivingAddress);
+      });
+      cy.findByTestId("Governance.Create.Step1.ReviewProposal").click();
+    });
+
+    it("should have step 2 header and description", () => {
```

Comment:
can we add a test to check that step 1 content is hidden before this test?

Reply:
added test to check that steps is hidden